app/models/account_credit_card.rb

```diff
@@ -0,0 +1,8 @@
+class AccountCreditCard < ApplicationRecord
+  belongs_to :account
+  belongs_to :owner, class_name: 'User', foreign_key: 'owner_id'
+
+  validates :bank, presence: true
+  validates :brand, presence: true
+  validates :credit_limit, numericality: { greater_than: 0}, presence: true
```

Comment:
I think it'd be nice to have presence validations on `owner_name` and `expired_at` too. _Maybe_ `last_digits`, but not sure, it's you call.

![image](https://user-images.githubusercontent.com/14297526/186904045-b70fddb4-918b-45ee-801a-11c50a798313.png)